### PR TITLE
add fieldFeaturedContent to "page" liquid template

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
- Added `fieldFeaturedContent` to page GraphQL query and liquid template

## Testing done
- Locally with staging data

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/page.graphql.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ module.exports = `
     title
     fieldIntroText
     fieldDescription
+    fieldFeaturedContent {
+      entity {
+        entityType
+        entityBundle
+        ${WYSIWYG}
```
Comment (jbalboni) [inline src/site/stages/build/drupal/graphql/page.graphql.js]: Is it worth adding the other items from `fieldContentBlock` here?

**Review by jbalboni — APPROVED** [review]
Looks good, just had one question

File: src/site/stages/build/drupal/graphql/page.graphql.js (hunk 1 above)
Comment (ahay-agile6) [reply src/site/stages/build/drupal/graphql/page.graphql.js]: as far as i know... no.
these can only be these two types, according to the drupal settings. i can double check with @beeyayjay 

on staging:
screen shot 2019-02-12 at 2 19 15 pm
the rest are unchecked as well

Comment (ahay-agile6) [reply src/site/stages/build/drupal/graphql/page.graphql.js]: @jbalboni @beeyayjay said that is correct - it can only be wysiwyg or q & a

**Pull request merged**